_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "Index management UI enhancements"
+authors:
+- ihailong
+- gbinlong
+- suzhou
+- lxuesong
+date: 2023-02-28
+categories:
+ - technical-post
+meta_keywords: index operations, split, shrink, reindex, index template, alias
+meta_description: Simplify cluster operations with index management UI enhancements.
```

Comment:
Please update the meta keywords and description with the following: 

**Meta_keywords:**  index management, index mappings, index operation UI, shrink index operations, OpenSearch Dashboards
**Meta_description:** Simplify cluster operations with index management UI enhancements that enable you to open, close, reindex, shrink, and split indexes in a more user-friendly way.

Reply:
Done for this